composer.json

```diff
@@ -16,11 +16,12 @@
     ],
     "require": {
         "php": ">=7.2",
+        "ext-intl": "*",
```

Comment:
Could you please remind me where it's being used? Maybe we could drop it, one way or another. Having less dependencies is always better :)

Reply:
Hmm, actually it's not in this package, it's in the `php-to-string` one. I'm going to make a PR there.

Reply:
Yeah, when I installed this package it had used `coduo/php-to-string` `2.0.1`, which doesn't have that `intl` requirement.